src/backend/distributed/deparser/ruleutils_11.c

```diff
@@ -7488,6 +7489,122 @@ pg_get_triggerdef_command(Oid triggerId)
 	return pg_get_triggerdef_worker(triggerId, prettyOutput);
 }
 
+char *
+pg_get_statisticsobj_worker(Oid statextid, bool missing_ok)
```

Comment:
I feel this function comes from postgres directly right?
Will not review it

Reply:
yes, copy-pasted